src/pipeline-server/docker-compose.pipeline-server.yml

```diff
@@ -17,10 +17,8 @@ services:
     ipc: "none"
     ports:
       - "8555:8554"
-    environment:
-      http_proxy: ${http_proxy}
-      https_proxy: ${https_proxy}
-      no_proxy: ${no_proxy}
+    env_file:
+      - .env
```

Comment:
The original code will get the proxy ENV settings directly from the host system. This makes the code easier to use by automatically providing the proxy settings for the user.